datastore/mock_database.go

```diff
@@ -483,6 +489,47 @@ func (mdb *MockDB) ListAccountUsers(authorityID string) ([]User, error) {
 	return mdb.ListUsers()
 }
 
+// CreateKeypairStatusTable mock the database creation
+func (mdb *MockDB) CreateKeypairStatusTable() error {
```

Comment:
Don't you dare giving a go (pun intended) to mock framework? https://godoc.org/github.com/golang/mock/gomock

Reply:
Not right now - there's quite a few other priorities. We have a task to change the tests, so it would be better to handle it there. These tests are an extension of existing ones, so it didn't feel like the right approach to take the newer approach.